Xenakios/TakeRenaming.cpp

```diff
@@ -223,40 +165,39 @@ void DoRenameTakeAndSourceFileDialog(COMMAND_T* ct)
 					if (newfnsplit[2].compare(fnsplit[2])!=0)
 						newfilename.append(fnsplit[2]);
 
-					MoveFile(oldname.c_str(),newfilename.c_str());
+					if (FileOrDirExists (newfilename.c_str()))
+					{
+						if (MessageBox(g_hwndParent, __LOCALIZE("Filename already exists or file could not be offlined.\nIf you continue it might create a wrong source reference in your REAPER project!\nContinue?", "sws_DLG_165"), __LOCALIZE("Xenakios - Warning", "sws_DLG_165"), MB_OKCANCEL) == IDCANCEL)
```

Comment:
Offline status would not affect `FileOrDirExists`. Shouldn't this message just mention that the file already exists and asks whether to overwrite it?